src/engine/renderer/tr_image.cpp

```diff
@@ -1453,7 +1458,10 @@ R_CreateImage
 */
 image_t *R_CreateImage( const char *name, const byte **pic, int width, int height, int numMips, const imageParams_t &imageParams )
 {
-	Log::Debug( "Creating image %s (%d×%d, %d mips)", name, width, height, numMips );
+	static const std::string colorspaces[] = { "linear", "sRGB" };
+	std::string colorspace = colorspaces[ bool( imageParams.bits & IF_SRGB ) ];
```

Comment:
Use a reference to avoid copying. (Or use `char*` instead of `string`)

Reply:
Done, used `char*`.